src/nvim/fold.c

```diff
@@ -762,6 +762,9 @@ void clearFolding(win_T *win)
  */
 void foldUpdate(win_T *wp, linenr_T top, linenr_T bot)
 {
+  if (disable_fold)
+	  return;
```

Comment:
can this check `compl_busy` instead (which will need to be moved to globals.h in some fashion)?